tests/utils.go

```diff
@@ -320,6 +335,48 @@ func BeforeTestSuitSetup() {
 	CreatePVC(osWindows, defaultWindowsDiskSize)
 }
 
+func GetListOfManifests(pathToManifestsDir string) []string {
+	var manifests []string
+	files, err := ioutil.ReadDir(pathToManifestsDir)
+	if err != nil {
+		panic(err)
+	}
+	matchFileName := func(pattern, filename string) bool {
+		match, err := filepath.Match(pattern, filename)
+		if err != nil {
+			panic(err)
+		}
+		return match
+	}
+	for _, file := range files {
+		if matchFileName("*-for-ocp.yaml", file.Name()) {
+			if CurrentClusterType.value == OpenShiftClusterType {
+				manifests = append(manifests, file.Name())
+			}
+		} else if matchFileName("*for-k8s.yaml", file.Name()) {
+			if CurrentClusterType.value == KubernetesClusterType {
+				manifests = append(manifests, file.Name())
+			}
+		} else if matchFileName("*.yaml", file.Name()) {
+			manifests = append(manifests, file.Name())
+		}
+	}
+	return manifests
+}
+
+func DeployTestingInfrastructure() {
+	fmt.Printf(fmt.Sprintf("Manifest dir: %s\n", PathToTestingInfrastrucureManifests))
+	manifests := GetListOfManifests(PathToTestingInfrastrucureManifests)
+	for _, manifest := range manifests {
+		// TODO: Load manifest and send it via AP
```

Comment:
@rmohr I have an ambitious Q: is there a way to post whole manifest via client-go (like oc create -f X) ?
Or do I need to read the manifest and split it per kinds, and call appropriate api call?

:-)

Reply:
@lukas-bednar I think you can unmarshal the whole manifest to a struct and create this struct via API

Reply:
Yes @cynepco3hahue someting like https://github.com/kubernetes/client-go/issues/193#issuecomment-343138889? For posting, I think you will have to do a switch-case statement.

Maybe we should simply start having all these manifests in go code from the begin with and just serialize them out during the build ... (not necessary as part of this pr).

Reply:
@rmohr @cynepco3hahue I see, I will go in this switch-case direction. Thanks.

Reply:
Hm, here we have two options:

1) copy over SecurityContextContextConstraints from https://github.com/openshift/origin/blob/master/pkg/security/apis/security/types.go#L19 (not so much a bad idea because whateve will change in origin, this is already v1 and will always work) and register it like it is done here: https://github.com/kubevirt/kubevirt/blob/master/pkg/api/v1/types.go#L92.
2) the rest client can do raw requests to arbitrary paths too. So instead of deserializing them first, simply send them directly.